EPPs/etc/customer_manifest.html

```diff
@@ -5,11 +5,15 @@
 </head>
 <body><div style="font-family:Calibri,Arial, sans-serif; font-size:12.0pt">
     <p> Dear <span style="color: #ff0000"> CUSTOMER NAME(S)</span>,</p>
-    <p>Thank you for choosing to use our sequencing service. I am the LIMS Manager at Edinburgh Genomics Clinical and I will liaise with you from sample submission up until the release of your data. I will reference this project using <b>{{ project }}</b> in the subject line of all future email correspondence.<br>
+    <p><span  style="color: #ff0000">Thank you for choosing to use our sequencing service. I am the Project Coordinator at Edinburgh Genomics Clinical and I will liaise with you from sample submission up until the release of your data. I will reference this project using <b>{{ project }}</b> in the subject line of all future email correspondence.<br>
         <br>Please find attached the following two documents:<br>
+        Thank you for choosing to use our sequencing service again. Please find attached the following two documents:<br>
```

Comment:
> Thank you for choosing to use our sequencing service again

Are we assuming this will always be a returning customer?

Reply:
The new project team has requested a range of text that covers most possible scenarios and they will delete what they don't need.

Reply:
I should add that the project team reviewed the template text prior to the pull request.